test/httpServer.cpp

```diff
@@ -366,116 +395,110 @@ void HttpServer::OnRequestUrlPost(mg_connection* conn, http_message* msg) {
                 "\n"
                 "}"};
     }
-    mg_send_head(conn, 201, response.length(), headers.c_str());
-    mg_send(conn, response.c_str(), response.length());
+    mg_http_reply(conn, 201, headers.c_str(), response.c_str());
 }
 
-void HttpServer::OnRequestBodyGet(mg_connection* conn, http_message* msg) {
-    if (std::string{msg->method.p, msg->method.len} != std::string{"GET"}) {
-        mg_http_send_error(conn, 405, "Method Not Allowed");
+void HttpServer::OnRequestBodyGet(mg_connection* conn, mg_http_message* msg) {
+    if (std::string{msg->method.ptr, msg->method.len} != std::string{"GET"}) {
+        std::string errorMessage{"Method Not Allowed"};
+        SendError(conn, 405, errorMessage);
         return;
     }
     std::array<char, 100> message{};
-    mg_get_http_var(&(msg->body), "message", message.data(), message.size());
+    mg_http_get_var(&(msg->body), "message", message.data(), message.size());
     if (msg->body.len <= 0) {
-        mg_http_send_error(conn, 405, "No Content");
+        std::string errorMessage{"No Content"};
+        SendError(conn, 405, errorMessage);
         return;
     }
     std::string response = message.data();
-    std::string headers = "Content-Type: text/html";
-    mg_send_head(conn, 200, response.length(), headers.c_str());
-    mg_send(conn, response.c_str(), response.length());
+    std::string headers = "Content-Type: text/html\r\n";
+    mg_http_reply(conn, 200, headers.c_str(), response.c_str());
 }
 
-void HttpServer::OnRequestJsonPost(mg_connection* conn, http_message* msg) {
+void HttpServer::OnRequestJsonPost(mg_connection* conn, mg_http_message* msg) {
     mg_str* content_type{nullptr};
-    if ((content_type = mg_get_http_header(msg, "Content-Type")) == nullptr || std::string{content_type->p, content_type->len} != "application/json") {
-        mg_http_send_error(conn, 415, "Unsupported Media Type");
+    if ((content_type = mg_http_get_header(msg, "Content-Type")) == nullptr || std::string{content_type->ptr, content_type->len} != "application/json") {
+        std::string errorMessage{"Unsupported Media Type"};
+        SendError(conn, 415, errorMessage);
         return;
     }
 
-    std::string headers = "Content-Type: application/json";
-    mg_send_head(conn, 201, msg->body.len, headers.c_str());
-    mg_send(conn, msg->body.p, msg->body.len);
+    std::string headers = "Content-Type: application/json\r\n";
+    mg_http_reply(conn, 201, headers.c_str(), msg->body.ptr);
 }
 
-void HttpServer::OnRequestFormPost(mg_connection* conn, http_message* msg) {
-    char var_name[100];
-    char file_name[100];
-    const char* chunk;
-    size_t chunk_len = 0;
-    size_t n1 = 0;
-    size_t n2 = 0;
+void HttpServer::OnRequestFormPost(mg_connection* conn, mg_http_message* msg) {
+    size_t pos{0};
+    struct mg_http_part part {};
```

Comment:
```suggestion
    struct mg_http_part part{};
```

Reply:
`clang-format -style=file -i test/httpServer.cpp` adds a space here. Should I leave the space there or remove it?

Reply:
Then we are keeping it there. But why doesn't it then add a space the line above?

Reply:
Without the `struct` keyword, the space is removed

Reply:
OK, interesting :D